.eslintrc

```diff
@@ -5,7 +5,8 @@
 		"max-len": "off",
 		"padded-blocks": "off",
 		"no-confusing-arrow": "off",
-		"no-underscore-dangle": "off"
+		"no-underscore-dangle": "off",
+		"linebreak-style": 0
```

Comment:
What is this fixing please?

Reply:
Here is a good explanation:
https://stackoverflow.com/questions/37826449/expected-linebreaks-to-be-lf-but-found-crlf-linebreak-style-in-eslint-using